ontology/uco-bfo.ttl

```diff
@@ -66,10 +66,18 @@ uco-core:Event
 		;
 	.
 
+uco-core:Facet
+	rdfs:subClassOf obo:BFO_0000020 ;
+	.
+
 uco-core:Item
 	rdfs:subClassOf drafting:Endurant ;
 	.
 
+uco-core:hasFacet
+	rdfs:subPropertyOf obo:BFO_0000196 ;
+	.
+
```

Comment:
`obo: BFO_0000196` - [bearer of](https://github.com/BFO-ontology/BFO-2020/blob/master/src/owl/bfo-core.ttl#L541)